### ISSUE TITLE
[Metrics Section]

### ISSUE THREAD

**Issue by mauricioarce** [swcbba/sw-website-2019#6]
- [ ] Versions
- [ ] Participants
- [ ] Projects
- [ ] Ideas

**Comment by xkris6x** [comment]
Is it ready to start?

**Comment by mauricioarce** [comment]
Please, send me first the PR related with the about section. You can't start a new issue, until the one you are working on has been finished. If you send me both in a single PR I will reject it.
